api/v1alpha1/spinappexecutor_types.go

```diff
@@ -56,6 +59,21 @@ type SpinAppExecutorStatus struct {
 	// Important: Run "make" to regenerate code after modifying this file
 }
 
+// OtelConfig is the supported environment variables for OpenTelemetry
+type OtelConfig struct {
+	// Sets OTEL_EXPORTER_OTLP_ENDPOINT environment variable
+	OtelExporterOtlpEndpoint string `json:"exporter_otlp_endpoint"`
+
+	// Sets OTEL_EXPORTER_OTLP_TRACES_ENDPOINT environment variable
+	OtelExporterOtlpTracesEndpoint string `json:"exporter_otlp_traces_endpoint"`
+
+	// Sets OTEL_EXPORTER_OTLP_METRICS_ENDPOINT environment variable
+	OtelExporterOtlpMetricsEndpoint string `json:"exporter_otlp_metrics_endpoint"`
+
+	// Sets OTEL_EXPORTER_OTLP_LOGS_ENDPOINT environment variable
+	OtelExporterOtlpLogsEndpoint string `json:"exporter_otlp_logs_endpoint"`
```

Comment:
```suggestion
	// ExporterOtlpEndpoint configures the default combined otlp endpoint for sending telemetry
	ExporterOtlpEndpoint string `json:"exporter_otlp_endpoint,omitempty"`

	// ExporterOtlpTracesEndpoint configures the trace-specific otlp endpoint
	ExporterOtlpTracesEndpoint string `json:"exporter_otlp_traces_endpoint,omitempty"`

	// ExporterOtlpMetricsEndpoint configures the metrics-specific otlp endpoint
	ExporterOtlpMetricsEndpoint string `json:"exporter_otlp_metrics_endpoint,omitempty"`

	// ExporterOtlpLogsEndpoint configures the logs-specific otlp endpoint
	ExporterOtlpLogsEndpoint string `json:"exporter_otlp_logs_endpoint,omitempty"`
```

this PR lgtm with a smol change to naming here - and updating comments to match go standards (starting the comment with the thing it's attached to's name).

Also needs a new `make manifests` committing because the current crd seems out-of-date.

Thanks for working on this!